ckanext/dcatapchharvest/tests/test_swiss_schemaorg_profile_serialize.py

```diff
@@ -184,3 +190,36 @@ def test_graph_from_dataset(self):
             eq_(len([t for t in g.triples((dataset_ref, item[1], None))]), len(values))
             for value in values:
                 assert self._triple(g, dataset_ref, item[1], item[2](value))
+
+    # new test
+    def test_graph_from_dataset_uri(self):
```

Comment:
It might be helpful to add a docstring here saying what the test is testing, e.g.

```
    def test_graph_from_dataset_uri(self):
    """"Tests that datasets with a uri from the test system have that uri changed to reference the prod system when they are output as a graph"""